flatpak/manifest_tool/tests/test_flutter_ops.py

```
@@ -541,3 +609,244 @@ def test_bundle_app_archive_no_lotti_module(make_document, tmp_path):
 
     # Should not change anything
     assert not result.changed
+
+
+# Tests for newly added functions
+
+
+def test_ensure_flutter_pub_get_offline(make_document):
```

Ruff (0.13.1)

Redefinition of unused `test_ensure_flutter_pub_get_offline` from line 106

(F811)